intro_to_rl.ipynb

```diff
@@ -0,0 +1,3249 @@
+{
```

Comment:
<p>Spelling grammar:</p><p> </p><p>RL algorithms will try to maximise the score in the game by <s>chosing</s> choosing the best actions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>